hello-thirdparty/app/build.gradle

```diff
@@ -1,9 +1,20 @@
 apply plugin: 'com.android.model.application'
 
-
 def gpg_sdk_path = file(project(':thirdparty:gpg-sdk').projectDir).absolutePath + "/gpg-cpp-sdk/android"
 
 model {
+
+    repositories {
+        libs(PrebuiltLibraries) {
+            gpg {
+                headers.srcDir "${gpg_sdk_path}/include"
+                binaries.withType(SharedLibraryBinary) { // NOTE: this block should be "StaticLibraryBinary"/staticLibraryFile - but SharedLibraryBinary works and StaticLibraryBinary doesn't as of 0.6.0-alpha1
```

Comment:
the feature wasn't available in 0.4. I've just tried with 0.6.0-alpha2 and get the same error:
:app:linkArmeabi-v7aDebugNative-activitySharedLibrary                 
org.gradle.nativeplatform.internal.prebuilt.PrebuiltLibraryResolveException: Shared library link file not set for prebuilt library 'gpg'.
        at org.gradle.nativeplatform.internal.prebuilt.AbstractPrebuiltLibraryBinary$ValidatingFileSet.getFiles(AbstractPrebuiltLibraryBinary.java:99)

I'll report it.